### PR TITLE
Purescript 0.13

### PR DESCRIPTION
**Pull request by jvliwanag**
Updates the generator project to purescript 0.13. This does not include the templated (and generated) projects yet.

I had to disable generation in the Makefiles for now. This should be restored in a future pull req. Let me know if it might be better to delay this pull req for now until everything works with 0.13.

### CODE REVIEW

File: wercker.yml [hunk 1]
```diff
@@ -30,13 +30,15 @@ build:
         name: build
         code: |
           echo Building project...
-          make clean init build run init-all
+          # Skip init-all first until templated project is upgraded to purescript 0.13
```
Comment (plippe) [inline wercker.yml]: Isn't this fixed now ?

**Comment by plippe** [comment]
I think it would be better to merge when everything is on 0.13

**Comment by jvliwanag** [comment]
Oh, sorry i didn’t reply earlier. Still working on other updates to make everything work with 0.13. I’ll open other pull reqs then for the other changes

**Comment by plippe** [comment]
I thought you were waiting on me ... communication is hard. 

Sorry if I merged too early

**Pull request merged**